client/src/pages/Prospects.jsx

```diff
@@ -1,9 +1,62 @@
-import React from 'react'
+import React, { useState } from "react";
+import Navbar from "../components/Navbar";
+import { makeStyles } from "@material-ui/core/styles";
+import ProspectsTable  from '../components/prospects/Table';
+import {Button, Grid, IconButton, Snackbar, Typography, Container} from "@material-ui/core";
 
-export default function Prospects(props) {
-  return(
-    <div>
-      Welcome to your Prospects!
+export default function Prospects() {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <Grid container spacing={3}>
+        <Grid item className={classes.sidebar} sm={2}>
+          Sidebar
+        </Grid>
+        <Grid item sm={10}>
+          <Container className={classes.contentContainer}>
+            <Grid item className={classes.contentHeader} sm={12}>
+              Content Header
+              <button onClick={Test}>
+                Click me!
+              </button>
+            </Grid>
+            <Grid item className={classes.content} sm={12}>
+              <ProspectsTable/>
+            </Grid>
+          </Container>
+        </Grid>
+      </Grid>
     </div>
   )
-}
\ No newline at end of file
+}
+
+function Test() {
+  fetch("http://localhost:5000/api/prospects", {
```

Comment:
You actually don't need to mention http://localhost:5000 because of the proxy set up in client/package/json

It should work if you just wrote fetch("/api/prospects")